src/app/teams/team-details/team-details.component.ts

```diff
@@ -3,8 +3,9 @@ import { ActivatedRoute, Params } from '@angular/router';
 
 import { TitleService } from 'app/title.service';
 import { ProfileService } from 'app/profile';
-import { Team, PlayerRole } from 'app/types';
+import {Team, PlayerRole, GameEvent} from 'app/types';
```

Comment:
Again. Spaces around import. As a matter of fact I like no-spaces too, but it seems to be more common in the angular world to have them